IFC4x3/Properties/g/GroundClassTBM_1_Vnw4BBXCOA6hWdCFm02c/Documentation.md

```diff
@@ -0,0 +1 @@
+Specific rock mass classification based on TBM data.
```

Comment:
Whenever I see classification I think of [Classification Association](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/classification-association.htm).

Reply:
I prefer to leave this with @j-weil. 
[Classification Association](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/classification-association.htm) would need a specific enough classification system since the classification association as such does not provide any specific semantics about the nature of the classification.